.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
+-   repo: https://github.com/ambv/black
+    rev: stable
+    hooks:
+    - id: black
+      language_version: python3.6
```

Comment:
Python 3.6 is not available on Ubuntu 16.04 LTS, which is supported
until April 2021. If you require Python 3.6+, I can’t build or run your
code. :-(